README.md

```diff
@@ -31,6 +31,15 @@ request][pr].
 
 See [CONTRIBUTING.md][].
 
+We (OTel Comms SIG) meet every two weeks on Thursdays at 10:30 PT.
+Check out the [OpenTelemetry community calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
+for the Zoom link and any updates to this schedule.
+
+Meeting notes are available as a public [Google
+doc](https://docs.google.com/document/d/1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY/edit?usp=sharing).
+If you have trouble accessing the doc, please get in touch on
```

Comment:
I am wondering if promoting slack as a place where we communicate async would be better than say it's for troubleshooting file access?

Reply:
Yeah, probably. So I made a terrible mistake in using the github editor here: I don't actually have permissions to edit my own branch on this repo. So I'm happy to close and re-open a different PR that augments this sentence 💀